modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -538,11 +538,11 @@ export default {
       const doc = await apos.http.get(`${action}/${this.context.aposDocId}`, {
         qs: {
           aposMode: this.draftMode,
-          project: { _url: 1 }
+          project: { slug: 1 }
         }
       });
 
-      const url = apos.http.addQueryToUrl(doc._url, qs);
+      const url = apos.http.addQueryToUrl(doc.slug, qs);
```

Comment:
Unlike `slug`, `_url`  contains the locale prefix, which results in having the final url constructed with a double locale prefix such as `/fr/fr/some-page`.

cc @haroun